.readthedocs.yml

```diff
@@ -5,9 +5,9 @@ version: 2
 # Configure the python version and environment construction run before
 # docs are built.
 build:
-  os: ubuntu-22.04
+  os: ubuntu-24.04
```

Comment:
I think the sphinx defaults are ubuntu 22.04 and py 3.11. Did you go higher than that just to be super up to date?

Reply:
What do you mean by the sphinx defaults? I just updated to the the most recent major releases of both.